src/commands/invest/portfolio/slash.ts

```diff
@@ -103,61 +131,42 @@ const slashCmd: SlashCommand = {
       }
     }
 
-    // no data
-    if (!data) {
-      return {
-        messageOptions: {
-          embeds: [
-            composeEmbedMessage2(i, {
-              title: "Your earning portfolio is empty",
-              description: `You have not invested in any earning platform yet.\n${getEmoji(
-                "ANIMATED_POINTING_RIGHT",
-                true
-              )}You can invest in earning platform by using ${await getSlashCommand(
-                "invest stake"
-              )}`,
-            }),
-          ],
-        },
-      }
-    }
+    const info = data
+      .map((invest: InvestBalance) => {
+        const record = []
+        const decimals = invest.to_underlying_token.decimals
+        const tokenAmount =
+          Number(invest.to_underlying_token?.balance) / 10 ** decimals
+        const amount = `${formatTokenDigit(tokenAmount)} ${
+          invest.to_underlying_token.symbol
+        }`
+        const apy = `${formatTokenDigit(invest.apy)}%`
+        const usdWorth = `$${formatUsdDigit(invest.underlying_usd)}`
+        const platform = invest.platform.name || ""
+        const symbol = invest.to_underlying_token?.symbol || ""
 
-    // data
-    const { segments } = formatDataTable(
-      [
-        {
-          chainID: "ChainID",
-          platform: "Platform",
-          token: "Token",
-          amount: "Amount",
-          apy: "APY",
-        },
-        ...data.map((invest) => {
-          const decimals = invest.to_underlying_token.decimals
-          const tokenAmount = (
-            Number(invest.to_underlying_token?.balance) /
-            10 ** decimals
-          ).toFixed(2)
-          return {
-            chainID: invest.chain_id,
-            platform: invest.platform.name ?? "NA",
-            token: `${invest.to_underlying_token.symbol}`,
-            amount: tokenAmount,
-            apy: `${invest.apy.toFixed(2)}%`,
-          }
-        }),
-      ],
-      {
-        cols: ["chainID", "platform", "token", "amount", "apy"],
-        separator: [VERTICAL_BAR],
-      }
-    )
+        record.push({
+          emoji: getEmoji(symbol as TokenEmojiKey),
+          amount,
+          usdWorth,
+          platform: platform,
+          apy: apy,
+        })
+        return record
+      })
+      .flat()
```

Comment:
sao cần xài array r flat ra chi em? sao ko return trực tiếp cái json luôn? 😆 

```suggestion
      return {
          emoji: getEmoji(symbol as TokenEmojiKey),
          amount,
          usdWorth,
          platform: platform,
          apy: apy,
        }
      })
```